### PR TITLE
Connect `LeftInverse` with (`Split`)`Surjection`

### PR DESCRIPTION
**Pull request by laMudri**
This PR partially addresses #1678. I have not decided what to do with `Function.Definitions` (potential confusion over its definition of `Surjective`) or `Epi` (because I'm not confident on which way to address the size issue is right).

### CODE REVIEW

File: src/Function/Structures.agda [hunk 1]
```diff
@@ -105,6 +105,22 @@ record IsLeftInverse (to : A → B) (from : B → A) : Set (a ⊔ b ⊔ ℓ₁ 
   strictlyInverseˡ : StrictlyInverseˡ _≈₂_ to from
   strictlyInverseˡ x = inverseˡ Eq₁.refl
 
+  isSurjection : IsSurjection to
+  isSurjection = record
+    { isCongruent = isCongruent
+    ; surjective = λ y → from y , inverseˡ
+    }
```
Comment (laMudri) [inline src/Function/Structures.agda:112]: One could argue for this to be called `to-isSurjection`. I don't know which name convention suggests.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2658,7 +2658,7 @@ Other minor changes
                 ∀ {m n} → _Respectsʳ_ (_<_ {m} {n}) _≋_
   <-wellFounded : Symmetric _≈_ →  Transitive _≈_ → _≺_ Respectsʳ _≈_ → WellFounded _≺_ →
                   ∀ {n} → WellFounded (_<_ {n})
-```
+  ```
```
Comment (laMudri) [inline CHANGELOG.md:2661]: This is a drive-by fix, but important because the formatting of the entire rest of the file is broken without it.

File: src/Function/Bundles.agda [hunk 3]
```diff
@@ -188,14 +188,54 @@ module _ (From : Setoid a ℓ₁) (To : Setoid b ℓ₂) where
...
+    isSplitSurjection = record { from = from ; isLeftInverse = isLeftInverse }
+
+    surjection : Surjection
+    surjection = record
+      { to = to
+      ; cong = to-cong
+      ; surjective = λ y → from y , inverseˡ
+      }
+
+  -- A left inverse is also known as a “split surjection”.
```
Comment (laMudri) [inline src/Function/Bundles.agda:209]: I don't know whether it's right to call an inhabitant of `LeftInverse` “a left inverse”.

**Comment by laMudri** [comment]
I forget how notifications work here, so maybe I should cc @mietek.

File: CHANGELOG.md (hunk 2 above)
Comment (jamesmckinna) [reply CHANGELOG.md:2661]: Is this fixed now?

Comment (MatthewDaggitt) [reply CHANGELOG.md:2661]: If it's not showing up as a conflict then it's probably not...

Comment (jamesmckinna) [reply CHANGELOG.md:2661]: Should it be lifted out as a separate 'PR' then (or directly edited on `master`)?

Comment (jamesmckinna) [reply CHANGELOG.md:2661]: Fixed in #2154

File: src/Function/Bundles.agda (hunk 3 above)
Comment (jamesmckinna) [reply src/Function/Bundles.agda:209]: No, but such usage seems to have been present since the old hierarchy (in `Function.Inverse`)...

File: src/Function/Structures.agda (hunk 1 above)
Comment (MatthewDaggitt) [reply src/Function/Structures.agda:114]: I think it's fine as it is.
